src/OrbitGl/LiveFunctionsDataView.cpp

```diff
@@ -93,10 +93,8 @@ std::string LiveFunctionsDataView::GetValue(int row, int column) {
       return GetPrettyTime(absl::Nanoseconds(stats.max_ns()));
     case kColumnModule:
       return function.module_path();
-    case kColumnAddress: {
-      const CaptureData& capture_data = app_->GetCaptureData();
-      return absl::StrFormat("0x%" PRIx64, capture_data.GetAbsoluteAddress(function).value_or(0));
-    }
+    case kColumnAddress:
+      return absl::StrFormat("%#" PRIx64, function.address());
```

Comment:
I think that we don't need `PRIx64` with `absl::StrFormat`, we can just use `"%#x"`.

Reply:
done.